crates/holochain/src/conductor/tests/install_app_bundle.rs

```diff
@@ -589,16 +593,12 @@ async fn use_existing_integration() {
         ));
     }
 
-    // Get the existing cell id through the normal means
-    let appmap = conductor
-        .cells_by_dna_lineage(dna1.dna_hash())
```

Comment:
I don't know why the cell was retrieved via `.cells_by_dna_lineage()` here but I don't think it's necessary for the purpose of the test and the function is not available anymore without the unstable-migration feature.